lib-es5/utils/index.js

```diff
@@ -1600,6 +1602,18 @@ function hashToQuery(hash) {
  */
 
 function present(value) {
+  // if (typeof value === 'string' || Array.isArray(value)) {
```

Comment:
Do we need that much of commented code left here?

Reply:
I left it by accident, updated in the recent commit.